packages/@react-aria/ssr/src/SSRProvider.tsx

```diff
@@ -79,6 +76,23 @@ export function SSRProvider(props: SSRProviderProps): JSX.Element {
   );
 }
 
+let warnedAboutSSRProvider = false;
+
+/**
+ * When using SSR with React Aria in React 16 or 17, applications must be wrapped in an SSRProvider.
+ * This ensures that auto generated ids are consistent between the client and server.
+ */
+export function SSRProvider(props: SSRProviderProps) {
```

Comment:
Is there a reason you removed the return type here? `): JSX.Element {` 
`props.children` would also be a JSX.Element, right?

Reply:
children can be a ReactNode, which could be a string, etc. in addition to an element. Anyway TS should auto-infer the type.